sewer/dns_providers/aliyundns.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python
```

Comment:
@menduo 
Please add a test for this class. 

Take a look at the tests for the various providers in; https://github.com/komuw/sewer/tree/master/sewer/dns_providers

Reply:
OK, done.